src/ParserConfigFile.cpp

```diff
@@ -320,6 +327,12 @@ void Raytracer::ParserConfigFile::parseCylindersInf(
       if (materialName == "reflective") {
         newCylinderInf->setMaterial(
             _factory.create<Raytracer::Reflections>("reflection"));
+      } else if (materialName == "transparent") {
+        newCylinderInf->setMaterial(
+            _factory.create<Raytracer::Transparency>("transparent"));
+      } else if (materialName == "refractive") {
+        newCylinderInf->setMaterial(
+            _factory.create<Raytracer::Refractions>("refraction"));
```

Comment:
```suggestion
      } else if (materialName == "refractive") {
        newCylinderInf->setMaterial(
            _factory.create<Raytracer::Refractions>("refraction"));
      } else if (materialName == "transparent") {
        newCylinderInf->setMaterial(
            _factory.create<Raytracer::Transparency>("transparent"));
```

j'ai juste inversé les 2 conditions pour rester cohérent avec les autres if/else